README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# hw01
+
+# Who am I?
+
+![](https://sociology.uchicago.edu/sites/sociology.uchicago.edu/files/styles/columnwidth-wider/public/uploads/images/Betsy%20Priem%20photo.jpg?itok=CiRQvArA)
+
+My name is *Betsy Priem* and I grew up in Minnesota. Former employment (dating back to early teenage years) includes:
+
+* Mower of neighbor's lawn
+* Gas station deli sandwich/pizza maker
+* Bank Teller
+* YMCA employee
+  + Youth tutor
+  + Tutor supervisor
+  + Student Board
+* Science teacher & Science content lead
+
+## Current research interests
+I'm currently pursuing a PhD in Sociology, and my main research interest lies in understanding the social aspects of the *climate crisis*. I look at how political parties evolved as more science became available on climate change, and the institutional processes that supported the parties ability to generate policies that would address or avoid the issue of climate change. Because of this, I often get labeled as a **political sociologist** or **environmental sociologist**. Further contact details can be found [here](https://sociology.uchicago.edu/directory/betsy-priem). 
```

Comment:
+Appreciated the overflow and look - starting with a bold "Who am I" was a great way to capture the attention of reader, followed by your photo
+The use of italics to emphasize your name was great and clear
+Use of linking contact info was a great way to end

Just a style thing, I would have made a new line for "Former employment..." that way your name and where you grew up stood on its own
Defining of linking what is a "political sociologist" or "environmental sociologist" would have been helpful/ appreciated
I'm not sure if the output/input box at the top of you document was on purpose? and if so - how did you do that and why did you include it?